tests/test_flip.py

```diff
@@ -55,15 +55,11 @@ def test_to_json_with_yaml(self):
 
     def test_to_json_with_json(self):
         """
-        Test that to_json still works when passed json
-        (All json is valid yaml)
+        Test that to_json fails when passed json
         """
 
-        actual = cfn_flip.to_json(self.input_json)
-
-        parsed_actual = json.loads(actual)
-
-        self.assertDictEqual(parsed_actual, self.parsed_json)
+        with self.assertRaises(ValueError):
+            actual = cfn_flip.to_json(self.input_json)
```

Comment:
Actually, we don't want this to happen. It shouldn't throw valid YAML input away when that YAML happens to also be valid JSON.